src/epics/sync.test.js

```diff
@@ -25,23 +24,44 @@ import getSyncEpic from '../epics/sync';
 import * as syncActionCreators from '../actions/syncActionCreators';
 import * as authActionCreators from '../actions/authActionCreators';
 
+const MOCK_ROUTES = [
```

Comment:
yes - I added it there and copied it here so I didn't have to wait for the package to publish, but it looks like greenkeeper has made the update so I can clean up this branch